### PR TITLE
Center Falco shine

### LINKED ISSUE #382: Falco shine not centered on his body

### PR DESCRIPTION
**Pull request by TeamPearson**
fixes #382

### CODE REVIEW

File: fighters/falco/src/acmd/specials.rs [hunk 1]
```diff
@@ -230,7 +210,7 @@ pub fn install() {
         falco_special_air_s_end_game,
         falco_special_hi_hold,
         falco_special_hi_game,
-        falco_special_lw_game,
+        // falco_special_lw_game,
```
Comment (jobrien97) [inline fighters/falco/src/acmd/specials.rs]: You can just delete this line rather than commenting out

**Review by jobrien97 — CHANGES_REQUESTED** [review]
Both falco_special_air_lw_game and falco_special_air_lw_r_game replace the same script, please delete one (preferably special_air_lw_r)
Also nitpick, but I'd like if you renamed falco_special_lw_r_game to just falco_special_lw_game, to avoid confusion. The script affects both left and right facing variants

File: fighters/falco/src/acmd/specials.rs (hunk 1 above)
Comment (TeamPearson) [reply fighters/falco/src/acmd/specials.rs]: This was the recommendation from blujay but I dont care to delete it.

**Review by jobrien97 — APPROVED** [review]
nice cleanup, lgtm

**Pull request merged**